root/static/scripts/common/components/TagEditor.js

```diff
@@ -360,20 +426,20 @@ function init_tag_editor(Component, mountPoint) {
         userTag.used = true;
       }
 
-      return Tag(t);
+      return newTag(t);
     });
 
     // Always show upvoted user tags (affects sidebar)
     _.each(userTags, function (t) {
       if (t.vote > 0 && !t.used) {
-        combined.push(Tag(t));
+        combined.push(newTag(t));
       }
     });
 
     ReactDOM.render(
       <Component entity={entity} more={more}
                  initialState={{tags: sortedTags(Immutable.List(combined))}} />,
-      document.getElementById(mountPoint)
+      (document.getElementById(mountPoint): any)
```

Comment:
`getElementById` can return `null`, but it shouldn't ever be `null` here, so I used `any` to silence the error.